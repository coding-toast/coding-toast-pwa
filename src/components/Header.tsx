import * as React from 'react'
import Navigation from './navigation/Navigation'

const Header: React.FC = () => <Navigation />

export default Header
