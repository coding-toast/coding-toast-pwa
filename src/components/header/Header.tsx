import * as React from 'react'
import Container, { BackgroundShade } from '../section-container/SectionContainer'
import QuoteBox from '../quote-box/QuoteBox'
import Logo from '../navigation/logo/Logo'
import NextSection from '../navigation/NextSection'
import LogoStyles from '../navigation/logo/Logo.module.scss'

const Header: React.FC = () => (
  <Container backgroundShade={BackgroundShade.PRIMARY} style={{ minHeight: '68vh' }}>
    <Logo styles={LogoStyles} />
    <QuoteBox />
    <NextSection />
  </Container>
)

export default Header
