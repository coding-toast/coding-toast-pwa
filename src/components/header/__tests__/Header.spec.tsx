import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Header from '../Header'

describe('<Header />', () => {
  it('renders correctly', () => {
    const component = shallow(<Header />)
    expect(toJson(component)).toMatchSnapshot()
  })
})
