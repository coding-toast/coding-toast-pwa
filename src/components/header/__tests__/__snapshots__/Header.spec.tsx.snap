// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<SectionContainer
  backgroundShade="#121212"
  style={
    Object {
      "minHeight": "68vh",
    }
  }
>
  <Logo
    styles={null}
  />
  <QuoteCard />
  <Memo(wrappedComponent) />
</SectionContainer>
`;
